test/jdk/javax/management/mxbean/SameObjectTwoNamesTest.java

```diff
@@ -41,8 +40,7 @@
 public class SameObjectTwoNamesTest {
 
     public static void main(String[] args) throws Exception {
-        boolean expectException =
-                (System.getProperty("jmx.mxbean.multiname") == null);
+        boolean expectException = true;
```

Comment:
I think it would be better to drop this variable at all (and remove unreachable code when the variable is "false")